src/subcommands/logs.ts

```diff
@@ -0,0 +1,160 @@
+// Copyright 2021 Deno Land Inc. All rights reserved. MIT license.
+
+import { wait } from "../../deps.ts";
+import { error } from "../error.ts";
+import { API, APIError } from "../utils/api.ts";
+
+const help = `deployctl logs
+Shows the logs of the given project.
+
+To show the latest logs of a project:
+  deployctl logs --project=helloworld
+
+To show the logs of a particular deployment:
+  deployctl logs --project=helloworld --deployment=1234567890ab
+
+To show the logs of the production deployment:
+  deployctl logs --project=helloworld --prod
+
+USAGE:
+    deployctl logs [OPTIONS] [<PROJECT>]
+
+OPTIONS:
+        --deployment=<DEPLOY_ID>  The id of the deployment you want to get the logs (defaults to latest deployment)
+        --prod                    Select the production deployment
+    -p, --project=NAME        The project you want to get the logs
+        --token=TOKEN             The API token to use (defaults to DENO_DEPLOY_TOKEN env var)
+`;
+
+export interface Args {
+  help: boolean;
+  prod: boolean;
+  token: string | null;
+  deployment: string | null;
+  project: string | null;
+}
+
+// deno-lint-ignore no-explicit-any
+export default async function (rawArgs: Record<string, any>): Promise<void> {
+  const args: Args = {
+    help: !!rawArgs.help,
+    prod: !!rawArgs.prod,
+    token: rawArgs.token ? String(rawArgs.token) : null,
+    deployment: rawArgs.deployment ? String(rawArgs.deployment) : null,
+    project: rawArgs.project ? String(rawArgs.project) : null,
+  };
+
+  if (args.help) {
+    console.log(help);
+    Deno.exit(0);
+  }
+  const token = args.token ?? Deno.env.get("DENO_DEPLOY_TOKEN") ?? null;
+  if (token === null) {
+    console.error(help);
+    error("Missing access token. Set via --token or DENO_DEPLOY_TOKEN.");
+  }
+  if (args.project === null) {
+    console.error(help);
+    error("Missing project ID.");
+  }
+  if (rawArgs._.length > 1) {
+    console.error(help);
+    error("Too many positional arguments given.");
+  }
+
+  const opts = {
+    project: args.project,
+    prod: args.prod,
+    token,
+    deployment: args.deployment,
+  };
+
+  await logs(opts);
+}
+
+interface DeployOpts {
+  project: string;
+  deployment: string | null;
```

Comment:
The names of these two field may be a little bit confusing, because these two actually refer to IDs, but there are also composite types named `Project` and `Deployment`:

```ts
export interface Deployment {
  id: string;
  url: string;
  domainMappings: DomainMapping[];
  project?: Project;
  projectId: string;
  createdAt: string;
  updatedAt: string;
  envVars: string[];
}

export interface Project {
  id: string;
  name: string;
  productionDeployment?: Deployment | null;
  hasProductionDeployment: boolean;
  organizationId: string;
  createdAt: string;
  updatedAt: string;
  envVars: string[];
}
```

So I would name these two fields as `projectId` and `deploymentId` to emphasize that they are IDs.